dashboard/db/schema.rb

```diff
@@ -350,6 +350,7 @@
     t.datetime "updated_at",                                                 null: false
     t.string   "type",                   limit: 255,   default: "Plc::Task", null: false
     t.text     "properties",             limit: 65535
+    t.string   "icon",                   limit: 255
```

Comment:
My thinking was that since this is a trait of all tasks, we should put it in the db.

But upon further consideration, we're never going to select on it. So yeah, it should be a serialized attribute